components/ShakeApi.js

```diff
@@ -0,0 +1,126 @@
+import React, { useEffect, useState } from 'react';
+import { View, Text, ActivityIndicator } from 'react-native';
+import styled from 'styled-components/native';
+import { Accelerometer } from 'expo-sensors';
+import { useFonts, Buda_300Light } from '@expo-google-fonts/buda';
+import { Poppins_400Regular } from '@expo-google-fonts/poppins';
+
+// Styled components
+
+const Heading = styled.Text`
+  font-size: 35px;
+  text-align: center;
+  margin-top: 10%;
+`;
+
+const TextBox = styled.View`
+  flex: 1;
+  justify-content: center;
+  align-items: center;
+  width: 90%;
+  margin: 0 auto;
+  margin-bottom: 40%;
+`;
+
+const QuoteText = styled.Text`
+  font-weight: 500;
+  font-size: 30px;
+  padding-bottom: 15px;
+`;
+
+const ShakeText = styled.Text`
+  font-size: 16px;
+  padding-right: 12%;
+`;
+
+const Spinner = styled.ActivityIndicator`
+  flex: 1;
+  justify-content: center;
+  align-items: center;
+`;
+
+const KanyeBackground = styled.ImageBackground`
+  height: 100%;
+`;
+
+const ShakeApi = () => {
```

Comment:
I didnt do a shake app, so i think it is so interesting to see how it is built up.